### PR TITLE
Move in-repo tests to use container image based storage

### PR DESCRIPTION
**Pull request by stuartwdouglas**
- Add option to disable localstack
- Default to image deployments if token is defined
- Configure image deployments in the e2e tests

### CODE REVIEW

File: openshift-with-appstudio-test/e2e/util.go
Comment (gabemontero) [inline openshift-with-appstudio-test/e2e/util.go:192]: should we reuse the `QUAY_USERNAME` already used in the dev flow?

**Review by gabemontero — CHANGES_REQUESTED** [review]
just one question

File: openshift-with-appstudio-test/e2e/util.go
Comment (stuartwdouglas) [reply openshift-with-appstudio-test/e2e/util.go:192]: Actually looking at the e2e repo I think this should probably be 'QUAY_E2E_ORGANIZATION'

**Comment by gabemontero** [comment]
the deploy and check for contaminants container had an image registry auth problem @stuartwdouglas : 
```
        2022-09-08 11:23:03,209 55df1004d27dabdc12e4db533fd222ed-build-0-task-pod quarkus-run.jar[1] ERROR [com.red.hac.sid.res.DeployResource] (executor-thread-1) Deployment failed: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry quay.io:443/redhat-appstudio-qe/test-images because: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:305)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:257)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticatePush(RegistryAuthenticator.java:190)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:336)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:326)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doPushBearerAuth(RegistryClient.java:308)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:66)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:35)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:131)
        	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:74)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:82)
        	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        	at java.base/java.lang.Thread.run(Thread.java:833)
        Caused by: com.google.cloud.tools.jib.http.ResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:355)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:266)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:283)
        	... 13 more
        Caused by: com.google.api.client.http.HttpResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.api.client.http.HttpResponseException$Builder.build(HttpResponseException.java:293)
        	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1118)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:349)
        	... 15 more
        
        2022-09-08 11:23:03,214 55df1004d27dabdc12e4db533fd222ed-build-0-task-pod quarkus-run.jar[1] ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (executor-thread-1) HTTP Request to /deploy failed, error id: 47499eab-27e9-40ca-a4f0-555415bfe204-1: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry quay.io:443/redhat-appstudio-qe/test-images because: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:305)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:257)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticatePush(RegistryAuthenticator.java:190)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:336)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:326)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doPushBearerAuth(RegistryClient.java:308)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:66)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:35)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:131)
        	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:74)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:82)
        	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        	at java.base/java.lang.Thread.run(Thread.java:833)
        Caused by: com.google.cloud.tools.jib.http.ResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:355)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:266)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:283)
        	... 13 more
        Caused by: com.google.api.client.http.HttpResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.api.client.http.HttpResponseException$Builder.build(HttpResponseException.java:293)
        	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1118)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:349)
        	... 15 more
        
        2022-09-08 11:23:03,215 55df1004d27dabdc12e4db533fd222ed-build-0-task-pod quarkus-run.jar[1] ERROR [org.jbo.res.rea.com.cor.AbstractResteasyReactiveContext] (executor-thread-1) Request failed: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry quay.io:443/redhat-appstudio-qe/test-images because: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:305)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:257)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticatePush(RegistryAuthenticator.java:190)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:336)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:326)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doPushBearerAuth(RegistryClient.java:308)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:66)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:35)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:131)
        	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:74)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:82)
        	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        	at java.base/java.lang.Thread.run(Thread.java:833)
        Caused by: com.google.cloud.tools.jib.http.ResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:355)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:266)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:283)
        	... 13 more
        Caused by: com.google.api.client.http.HttpResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.api.client.http.HttpResponseException$Builder.build(HttpResponseException.java:293)
        	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1118)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:349)
        	... 15 more
        
        2022-09-08 11:23:03,225 55df1004d27dabdc12e4db533fd222ed-build-0-task-pod quarkus-run.jar[1] ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (executor-thread-1) HTTP Request to /deploy/result failed, error id: 47499eab-27e9-40ca-a4f0-555415bfe204-2: java.lang.RuntimeException: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry quay.io:443/redhat-appstudio-qe/test-images because: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.redhat.hacbs.sidecar.resources.DeployResource.contaminates(DeployResource.java:76)
        	at com.redhat.hacbs.sidecar.resources.DeployResource$quarkusrestinvoker$contaminates_4b5df3f4054473d77c2829c2f70c6a3d2ba79fd4.invoke(Unknown Source)
        	at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
        	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:108)
        	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:140)
        	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:557)
        	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
        	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
        	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
        	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
        	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        	at java.base/java.lang.Thread.run(Thread.java:833)
        Caused by: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry quay.io:443/redhat-appstudio-qe/test-images because: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:305)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:257)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticatePush(RegistryAuthenticator.java:190)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:336)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:326)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doPushBearerAuth(RegistryClient.java:308)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:66)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:35)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:131)
        	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:74)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:82)
        	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        	... 1 more
        Caused by: com.google.cloud.tools.jib.http.ResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:355)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:266)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:283)
        	... 13 more
        Caused by: com.google.api.client.http.HttpResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.api.client.http.HttpResponseException$Builder.build(HttpResponseException.java:293)
        	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1118)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:349)
        	... 15 more
        
        2022-09-08 11:23:03,225 55df1004d27dabdc12e4db533fd222ed-build-0-task-pod quarkus-run.jar[1] ERROR [org.jbo.res.rea.com.cor.AbstractResteasyReactiveContext] (executor-thread-1) Request failed: java.lang.RuntimeException: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry quay.io:443/redhat-appstudio-qe/test-images because: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.redhat.hacbs.sidecar.resources.DeployResource.contaminates(DeployResource.java:76)
        	at com.redhat.hacbs.sidecar.resources.DeployResource$quarkusrestinvoker$contaminates_4b5df3f4054473d77c2829c2f70c6a3d2ba79fd4.invoke(Unknown Source)
        	at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
        	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:108)
        	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:140)
        	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:557)
        	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
        	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
        	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
        	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
        	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        	at java.base/java.lang.Thread.run(Thread.java:833)
        Caused by: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry quay.io:443/redhat-appstudio-qe/test-images because: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:305)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:257)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticatePush(RegistryAuthenticator.java:190)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:336)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:326)
        	at com.google.cloud.tools.jib.registry.RegistryClient.doPushBearerAuth(RegistryClient.java:308)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:66)
        	at com.google.cloud.tools.jib.builder.steps.AuthenticatePushStep.call(AuthenticatePushStep.java:35)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:131)
        	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:74)
        	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:82)
        	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        	... 1 more
        Caused by: com.google.cloud.tools.jib.http.ResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:355)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:266)
        	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:283)
        	... 13 more
        Caused by: com.google.api.client.http.HttpResponseException: 401 UNAUTHORIZED
        GET https://quay.io/v2/auth?service=quay.io&scope=repository:redhat-appstudio-qe/test-images:pull,push
        {"errors":[{"code":"UNAUTHORIZED","detail":{},"message":"Invalid Username or Password"}]}
        
        	at com.google.api.client.http.HttpResponseException$Builder.build(HttpResponseException.java:293)
        	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1118)
        	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:349)
        	... 15 more
```

first blush seems related to your PR here and using container image based storage.

**Comment by stuartwdouglas** [comment]
I think that failure is actually masking some kind of IOException.

**Comment by stuartwdouglas** [comment]
Auth failure was because this is an Oauth token and not a standard Base64 encoded username:password pair like local docker config uses. I have updated the code to attempt to deal with OAuth correctly.

**Comment by stuartwdouglas** [comment]
So it turns out that Jib can't handle an access token, it either needs a standard username:password combo (like in dockerconfig), or an OAuth refresh token (which it uses to get an access token). 

We seem to have an actual access token, so as a way to unblock this I have just directly hacked the access token into Jib using bytecode manipulation.

**Comment by stuartwdouglas** [comment]
Actually I just realised that QUAY_TOKEN is actually not a token, but a .dockerconfig file. I will update the code accordingly

**Comment by stuartwdouglas** [comment]
This is looking much more promising. It looks like the builds are completing and deploying, but the shaded build does not resolve the contaminate, so there is likely an issue with actually using the deployed containers.

**Comment by stuartwdouglas** [comment]
@gabemontero this is finally green, but I figure it is worth doing a service registry build before merging

/test jvm-build-service-in-repo-apicurio-e2e

**Review by gabemontero — APPROVED** [review]
/lgtm 
@stuartwdouglas 

feel free to merge based on what you see in the service registry results

won't get to it my Friday evening but can take a peek my Saturday AM

**Comment by gabemontero** [comment]
the initial apicurio build is still failing because of that dependency problem we discussed this past week in slack @stuartwdouglas @dwalluck 

`[ERROR] Failed to execute goal on project apicurio-registry-system-tests: Could not resolve dependencies for project io.apicurio:apicurio-registry-system-tests:jar:2.2.6-SNAPSHOT: Could not find artifact io.apicurio:apicurio-registry-operator-api-model:jar:1.1.0-SNAPSHOT in mirror.default (http://localhost:2000/maven2) -> [Help 1]`

Full log at https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-appstudio_jvm-build-service/230/pull-ci-redhat-appstudio-jvm-build-service-main-jvm-build-service-in-repo-apicurio-e2e/1568387004838711296/artifacts/jvm-build-service-in-repo-apicurio-e2e/gather-extra/artifacts/pods/jvm-build-service-test-namespace-7p9c5_service-registry-build-wc6d5-maven-run-pod_step-mvn-goals.log

I'm assuming that your changes here were not trying to address that @stuartwdouglas but I'll defer to you to merge this just in case.

**Comment by stuartwdouglas** [comment]
That is a different failure to the other one, the last one was a missing json artifact. This is a genuine failure from upstream (the repo fails to build locally without any of our stuff).

I think we need to target this job at a known good tag.

**Pull request merged**
